rti/mosaic-rti-api/src/main/java/org/eclipse/mosaic/rti/api/federatestarter/ExecutableFederateExecutor.java

```diff
@@ -63,7 +64,12 @@ public ExecutableFederateExecutor(FederateDescriptor descriptor, String command,
     public Process startLocalFederate(File workingDir) throws FederateStarterException {
         log.debug("start: (command={} {})", command, StringUtils.join(args, " "));
         try {
-            List<String> commandWithArgs = Lists.newArrayList(command);
+            List<String> commandWithArgs = new ArrayList<>();
+            if (command.endsWith(".bat")) {
```

Comment:
Is this really required? Shouldn't it be sufficient to define in the ambassador that the command is `cmd.exe` and the parameters are `"/C", "script.bat", ...` ?

Reply:
You are right. I reverted the commit and moved it to the ambassador.